PINNED_BZ_VERSION

```diff
@@ -0,0 +1,2 @@
+9.0.0.749
+https://web.archive.org/web/20231127163613/https://secure.backblaze.com/win32/install_backblaze.exe
```

Comment:
We should probably create a new snapshot before we release the first version with the autoupdater, to get the latest available included in the new docker (and so nobobdy, that used the old manual way to update can be on a version greater than the currently determined stable by us, to make the transition as easy ad possible).